packages/editor/src/client/panel/peerEditPanel.tsx

```diff
@@ -209,6 +210,19 @@ export function PeerEditPanel({id, onClose, onSave}: ImageEditPanelProps) {
               <DescriptionListItem label={t('peerList.panels.themeColor')}>
                 {profile?.themeColor}
               </DescriptionListItem>
+              <DescriptionListItem label={t('peerList.panels.callToActionText')}>
+                {!!profile?.callToActionText && (
+                  <RichTextBlock
+                    disabled
+                    // TODO: remove this
```

Comment:
Is this TODO still open?

Reply:
Nop, I am leaving it until we make the onChange prop not required. I am adding it in another PR